### PR TITLE
reorganize transfer pages

### PR DESCRIPTION
**Pull request by olehyev138**
Reorganize Transfer page and update styles

### CODE REVIEW

File: app/controllers/dashboard/transfers_controller.rb [hunk 1]
```diff
@@ -49,14 +49,6 @@ def fetch_chart_data
     render partial: 'chart'
   end
 
-  def prioritize
```
Comment (Alexey1100) [inline app/controllers/dashboard/transfers_controller.rb:52]: We are using this method, pls put it back : )

**Review by Alexey1100 — CHANGES_REQUESTED** [review]

File: app/controllers/dashboard/transfers_controller.rb (hunk 1 above)
Comment (olehyev138) [reply app/controllers/dashboard/transfers_controller.rb]: done

File: app/views/dashboard/transfers/_hot_wallet_mode.html.erb [hunk 2]
```diff
@@ -14,6 +14,7 @@
       <%= select_tag "project[hot_wallet_mode]",
                      options_for_select(Project.hot_wallet_modes.keys.map{|name| [name.humanize, name]}, project.hot_wallet_mode),
                      id: "project_hot_wallet_mode",
+                     style: 'border-left: none !important; padding-right: 45px; width: max-content;',
```
Comment (Alexey1100) [inline app/views/dashboard/transfers/_hot_wallet_mode.html.erb:17]: Pls put the styles into css file.

File: app/views/dashboard/transfers/_transfer.html.erb [hunk 3]
```diff
@@ -18,7 +18,13 @@
...
 
     <% if transfer.account %>
-      <%= render partial: "account", locals: {account: transfer.account} %>
+      <% sender_wallet_address = nil %>
+      <% sender_wallet_url = nil %>
+      <% if transfer.paid? %>
+        <% sender_wallet_address = transfer.blockchain_transactions.succeed.last&.source %>
+        <% sender_wallet_url = sender_wallet_address && transfer.token ? transfer.token.blockchain.url_for_address_human(sender_wallet_address) : nil %>
+      <% end %>
+      <%= render partial: "account", locals: {account: transfer.account, wallet: sender_wallet_address, wallet_url: sender_wallet_url} %>
```
Comment (Alexey1100) [inline app/views/dashboard/transfers/_transfer.html.erb:27]: Looks like there's a confusion.

`transfer.account` is the recipient not the sender, so we are rendering here the recipient account but with wallet address of the sender.

**Review by Alexey1100 — CHANGES_REQUESTED** [review]
Incorrect transfer recipient wallet address.

Also looks like this PR has some conflicts with #2107.
Could you merge them in one PR, so it would be easier to review and debug : )

**Comment by aquabu** [comment]
This PR is merged or related code is resolved by https://github.com/CoMakery/comakery-server/pull/2138

**Pull request closed without merging**